src/simulation/dynamics/spinningBodies/spinningBodiesNDOF/_UnitTest/test_spinningBodyNDOFStateEffector.py

```diff
@@ -0,0 +1,845 @@
+# ISC License
+#
+# Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
update copyright year to 2024 to be consistent with the other new files

Reply:
Done.